### ISSUE TITLE
invisible, non-damaging botflies are very confusing

### ISSUE THREAD

**Issue by wil-langford** [ganymedes01/TheErebus#564]
(TL;DR below)

I went to the Erebus and got a botfly larva attached. Violence wasn't working to detach it and that's all I'm good at.  I used /home to get back to the Overworld so that I could die with dignity surrounded by my beloved cows.  The botfly was invisible after I used /home, but that didn't stop it from killing me and activating my White Stone (from Random Things).  Then it killed me again.

When I awoke, I was no longer taking damage, so I recovered my gear, read up on botfly larvae so I'd know how to survive next time, and walked into the portal again.

Nothing happened.

I jumped up and down a bit.  Crouched.  Walked out and back in.

Nothing.

I removed the keystone, replaced the leaves, replaced the staff, and stepped into the reactivated portal.

Still nothing.

I checked the client logs, which didn't have anything to say about my attempts to hop dimensions.    Finally, I remembered seeing in passing that botfly larvae prevent portal usage.  At about the same time, I started to take damage from hunger, even though it hadn't been long enough for my full hunger and full saturation to wear off while standing still.

I checked, and I was affected by what looked suspiciously like botfly-larva-inflicted status effects.  I recalled that Beetle Juice was required, but I couldn't think of a way to get any without revisiting the Erebus.  In my hour of need, the server admin was off selfishly doing something unrelated, like caring for his infant son.

Eventually, a fluid cow was found that could provide me with much needed relief, and the portal works for me again.  I now keep a Beetle Juice cow in my yard and a bucket of the stuff handy whenever I start poking around the Erebus.

11/10.  Would be really confused again.  :)

======----======

TL;DR:  When you use /home to return from the Erebus with a botfly larva attached, it does not visibly render on return to the overworld, although it still deals damage.  After death, the botfly remains attached, but no longer deals damage directly.  Instead, it occasionally inflicts its status effects and triggers the larva issue that prevents portal usage.

Until the fix is in for the larva/portal issue, perhaps a way of obtaining Beetle Juice in the overworld (that doesn't rely on modded Fluid Cows being around) could be added, along with some sort of status indicator of botfly larva presence?

(I wrote the bug like I did both to work through my minor frustration and also to provide some searchable key words for others experiencing this problem.)

**Comment by vadis365** [comment]
Closing - 1.7.10 is no longer supported
